packages/neovim/build.sh

```diff
@@ -40,13 +38,6 @@ termux_step_host_build() {
 	chmod +x "$TERMUX_ORIGINAL_CMAKE"
 	export TERMUX_ORIGINAL_CMAKE="$TERMUX_ORIGINAL_CMAKE.orig"
 
-	mkdir -p $TERMUX_PKG_HOSTBUILD_DIR/deps
-	cd $TERMUX_PKG_HOSTBUILD_DIR/deps
-	cmake $TERMUX_PKG_SRCDIR/third-party
```

Comment:
I think this should not be removed. See https://github.com/termux/termux-packages/commit/2c77b5406c9b6e8ff056f266544df336439ca6e5.